src/ansys/fluent/core/solver/flobject.py

```diff
@@ -234,9 +234,14 @@ def get_state(self) -> StateT:
         """Get the state of the object."""
         return self.to_python_keys(self.flproxy.get_var(self.path))
 
-    def set_state(self, state: StateT):
+    def set_state(self, state: StateT = None, **kwargs):
         """Set the state of the object."""
-        return self.flproxy.set_var(self.path, self.to_scheme_keys(state))
+        if state:
```

Comment:
Is it necessary to check `state` if we are going to end up doing the same in the else block? Isn't `kwargs` the only check to do?